android/app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,9 @@
         <item name="colorPrimary">@color/primaryColor</item>
         <item name="colorPrimaryDark">@color/primaryColorDark</item>
         <item name="android:textColor">#000000</item>
+        <!-- From `expo-splash-screen` setup instructions. -->
+        <item name="android:drawable">@color/splashscreen_background</item>
+        <item name="android:windowBackground">@drawable/splashscreen</item>
```

Comment:
Hmm -- what if instead of `@drawable/splashscreen`, you wrote here `@color/splashscreen_background`? (Or `@color/splashscreenBackground`, after renaming.) Seems like that's all that `@drawable/splashscreen` consists of... and it itself is invoking the color from an attribute called `android:drawable`, so that suggests that a color counts as a drawable.

The Expo instructions need the extra indirection because the drawable XML is where you manually insert the image, if you so choose (with the `SplashScreenImageResizeMode.NATIVE` option.)

Reply:
Ooh, and it seems like we could remove splashscreen.xml altogether—yes, just tested, and that works.